src/ensembl/src/styles/_mixins.scss

```diff
@@ -0,0 +1,16 @@
+$selectedSpeciesBorderRadius: 20px;
+$selectedSpeciesPadding: 6px 20px;
+$deleteButtonPadding: 8px 20px 8px 28px;
+
+@mixin selected-species-common {
```

Comment:
I was thinking more in terms of a mixin that's the same for RoundButton, SelectedSpecies, and whatever it is we'll call this component:

![image](https://user-images.githubusercontent.com/6834224/96113539-36ce0f00-0edc-11eb-92a4-56d0cf5360d8.png)

If that is the case, the mixin shouldn't be called `selected-species-common`, but rather by the shape that it describes (`lozenge`?).

We probably need to confirm with Andrea that SelectedSpecies and RoundButton have the same shape. Perhaps their height is different, in which case the mixin can accept a `height` parameter.

Reply:
yes agreed. I will change that name to lozenge